conseil-lorre/src/main/resources/application.conf

```diff
@@ -30,6 +30,14 @@ lorre {
     }
   ]
 
+  # feature flags: select if some functionality should be turned off
+  enabled-features {
+    block-rights-fetching-is-on: true
+    block-rights-fetching-is-on: ${?CONSEIL_LORRE_BLOCK_RIGHTS_FETCHING_ENABLED}
+    fork-handling-is-on: true
+    fork-handling-is-on: ${?CONSEIL_LORRE_FORK_DETECTION_ENABLED}
```

Comment:
can you add this to docker-compose in the comment?

Reply:
Sure, thanks

Reply:
Done that, is that needed anywhere else?

Reply:
no, I dont think so